### ISSUE TITLE
vimium-ff compatability

### ISSUE THREAD

**Issue by frankieandone** [codebicycle/videospeed#232]
Can we add support for vimium extension so that there are no key conflicts with vimium-ff?

Proposal: use mnemonic keymaps on top of a setting to enable this feature (default: disabled) for backwards compat and interop for other users' configs.

**Comment by hmmmIndeed** [comment]
Yeah I would like that too but it doesn't seem like this extension will be updated any time soon given that it's been 4 years since there was any update.

**Comment by VladSavitsky** [comment]
Could confirm it conflicts with 'Vimium' extension (https://github.com/philc/vimium). I changed shortcuts from 'SD' to 'AS'.
